### PR TITLE
Feature/Remove-Merge outdated switch module 

### PR DESCRIPTION
**Pull request by abprasa**
# Description
The existing code has a list of switch modules defined based on the switch series, this new code removes/merges such existing outdated switch modules.

# Impacts
Making it switch type list clean in the UI and suitable for users to choose modules based on the version of the switch instead of the series. 

# NEW Package(s) required
No additional packages are needed. 

# Feature
This new code removes/merges the existing outdated switch modules defined based on the switch series, making it switch type list in UI clean and for a better user experience.

# Delete branch after the merge
YES 

# Checklist
- [ ] Document the feature In the Network Device Config Guide
- [ ] Migration Script
- [ ] Test the new modules on UI

# NEWS file entries
## New Features
* Removing/Merging existing switch model defined based on switch series for the next release

### CODE REVIEW

**Comment by fdurand** [comment]
There is still references of the cisco switches you remove in constants/switch_acls.pm

**Comment by abprasa** [comment]
> There is still references of the cisco switches you remove in constants/switch_acls.pm
Removed the references

**Comment by fdurand** [comment]
It miss the upgrade script in order to rename the switch module in the switches.conf (the one defined in use base)

**Comment by abprasa** [comment]
> It miss the upgrade script in order to rename the switch module in the switches.conf (the one defined in use base)

Created and updated the script

File: addons/upgrade/to-13.2-convert-switch-types.pl [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+=cut
+
+use strict;
+use warnings;
+use lib qw(/usr/local/pf/lib);
+use lib qw(/usr/local/pf/lib_perl/lib/perl5);
+use pf::IniFiles;
+use NetAddr::IP;
+use pf::util qw(valid_ip valid_mac run_as_pf);
```
Comment (fdurand) [inline addons/upgrade/to-13.2-convert-switch-types.pl]: valid_ip and valid_mac are useless

File: addons/upgrade/to-13.2-convert-switch-types.pl [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+use pf::file_paths qw(
+    $switches_config_file
+);
+use File::Copy;
```
Comment (fdurand) [inline addons/upgrade/to-13.2-convert-switch-types.pl]: useless

File: addons/upgrade/to-13.2-convert-switch-types.pl [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+print "Nothing to be done\n";
+
+=head1 AUTHOR
+
+Inverse inc. <info@inverse.ca>
+
+=head1 COPYRIGHT
+
+Copyright (C) 2005-2023 Inverse inc.
```
Comment (fdurand) [inline addons/upgrade/to-13.2-convert-switch-types.pl]: 2024

**Review by fdurand — CHANGES_REQUESTED** [review]

**Pull request merged**